docs/guides/basics/parameters/autocomplete.md

```diff
@@ -118,6 +118,11 @@ coda.makeParameter({
     },
     ```
 
+!!! bug "All parameters passed as string"
```

Comment:
After https://github.com/coda/coda/pull/72397 this should no longer be true 😄 , if you want to hold off on this one.